lms/static/js/edxnotes/views/tabs/course_structure.js

```diff
@@ -4,10 +4,12 @@ define([
     'gettext', 'underscore', 'js/edxnotes/views/note_group', 'js/edxnotes/views/tab_panel',
     'js/edxnotes/views/tab_view'
 ], function (gettext, _, NoteGroupView, TabPanelView, TabView) {
+    var view = "Location in Course";
     var CourseStructureView = TabView.extend({
+        view: 'Location in Course',
         PanelConstructor: TabPanelView.extend({
             id: 'structure-panel',
-            title: 'Location in Course',
+            title: view,
```

Comment:
Note that here is where the string used to be-- I just moved it up into a variable named "view" because I use it in 2 places (and I actually want the non-translated version for eventing).